docs/cli/modify-entity-line-item-change-dates-command.md

```diff
@@ -31,12 +31,6 @@ For the full list of options please refer to the helper option:
 $ sudo -u www-data bin/console roster:modify-entity:line-item:change-dates -h
 ```
 
-## Related environment variables
-
-| Variable | Description |
-|----------|-------------|
-| `DATABASE_URL` | Database connection string. Supported formats are described [here](https://www.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/configuration.html#connecting-using-a-url). |
-
```

Comment:
Why did you remove this? Any particular reason?

Reply:
Well, DATABASE_URL is related to the whole application, it was probably included here by mistake. Can't see why mention it here.

Reply:
Will simply put it back for now, tought it was added by me by mistake, didn't saw it was already spread everywhere.